src/app/style.rs

```diff
@@ -149,6 +153,116 @@ pub fn get_colored_table_row<'a>(
 						highlight_style,
 					));
 				// Colorize inside the arrows.
+
+				// FIXME: These need updating for breakage
+				} else if let (
+					Some(first_parenthesis),
+					Some(second_parenthesis),
+				) = (data.rfind('('), data.rfind(')'))
+				{
+					let inner = line[first_parenthesis..].to_string();
+
+					log::trace!(target: "style", "inner: {inner:?}");
+
+					// panic!("{:?}", data);
+
+					let expected = [
+						// expired
+						String::from("exp"),
+						// revoked
+						String::from("rev"),
+						// disabled
+						String::from("d"),
+						// invalid
+						String::from("i"),
+					];
+
+					let matched: Vec<_> = data.match_indices("(").collect();
+					// panic!("{:?}", matched);
+
+					// let s = String::from("foo rev bar");
+					// let matched: Vec<_> = s.match_indices("(").collect();
+
+					// panic!(
+					// 	"Data: {:?}, Inner: {:?}, Contains? {:?}",
+					// 	&s,
+					// 	&inner,
+					// 	s.contains(&inner)
+					// );
+
+					if let Some((opening_parenthesis, char)) =
+						data.match_indices("(").next()
+					{
+						let inner = data[(opening_parenthesis + 1)
+							..(opening_parenthesis + 4)]
+							.to_string();
+
+						// THIS WORKS
+						// panic!(
+						// 	"Data: {:?}, Inner: {:?}, Contains? {:?}",
+						// 	&data,
+						// 	&inner,
+						// 	expected.contains(&inner)
+						// );
+
+						if [
+							// expired
+							String::from("exp"),
+							// revoked
+							String::from("rev"),
+							// disabled
+							String::from("d"),
+							// invalid
+							String::from("i"),
+						]
+						.contains(&inner)
+						{
+							colored_line.push(Span::styled(
+								data[..first_parenthesis].to_string(),
+								highlight_style,
+							));
+							colored_line.push(Span::styled(
+								"(",
+								TuiStyle::default().fg(Color::DarkGray),
+							));
+							colored_line.push(Span::styled(
+								data[first_parenthesis + 1..second_parenthesis]
+									.to_string(),
+								TuiStyle::default().fg(Color::Red),
+							));
+							colored_line.push(Span::styled(
+								")",
+								TuiStyle::default().fg(Color::DarkGray),
+							));
+							colored_line.push(Span::styled(
+								data[second_parenthesis + 1..].to_string(),
+								highlight_style,
+							));
+						}
+					}
+
+				// colored_line.push(Span::styled(
+				// 	data[..first_parenthesis].to_string(),
+				// 	highlight_style,
+				// ));
+				// colored_line.push(Span::styled(
+				// 	"{",
+				// 	TuiStyle::default().fg(Color::DarkGray),
+				// ));
+				// colored_line.push(Span::styled(
+				// 	data[first_parenthesis + 1..second_parenthesis]
+				// 		.to_string(),
+				// 	TuiStyle::default().fg(Color::Cyan),
+				// ));
+				// colored_line.push(Span::styled(
+				// 	"}",
+				// 	TuiStyle::default().fg(Color::DarkGray),
+				// ));
+				// colored_line.push(Span::styled(
+				// 	data[second_parenthesis + 1..].to_string(),
+				// 	highlight_style,
+				// ));
+				// FIXME
```

Comment:
Looks like this needs a bit of cleanup :)

Reply:
Oops, thanks - sorry I wasn't working on this "professionally"